### ISSUE TITLE
Unable to open file or folder with keyboard

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/server#10008]
Currently you can navigate the file list via keyboard, with 3 Tab-clicks per row:
- Folder/file itself
- Sharing that folder/file
- 3-dot menu of that folder/file

Now the problem is that pressing enter on the folder/file itself opens the right sidebar instead of opening it. We need to change it to opening, otherwise it is not possible to enter into folders or open files.

@nextcloud/designers @nextcloud/accessibility

**Comment by skjnldsv** [comment]
@jancborchardt didn't we said we'll add this for 15?
With the new files app?

**Comment by jancborchardt** [comment]
@skjnldsv since changing this is such a small change but a really important accessibility fix to use files at all, we should absolutely do it for 14.

I mean, I assume it’s a small change cause it’s just switching from opening sidebar to opening the file.

**Comment by jancborchardt** [comment]
Also related is what @kevgathuku mentioned in https://github.com/nextcloud/server/issues/9994#issuecomment-400292143

**Comment by tyrylu** [comment]
Of course, i would be grateful if it could be done for 14.

**Comment by jancborchardt** [comment]
@tyrylu the fix is in https://github.com/nextcloud/server/pull/10310, if you want to review :) It will make it into Nextcloud 14! :tada:

**Comment by tyrylu** [comment]
Thank you, i'll look into it, at least i hope that i will have some time.
